Design Patterns/Design Patterns/Interfaces/IAbstractWheel.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace AbstractFactory.Interfaces
+{
+    public interface IAbstractWheel
```

Comment:
Such naming is weird for interfaces, word abstract is used gor classes, I thing IWheel or IEngine is better

Reply:
Fixed naming and some folder structures